client/my-sites/checkout/composite-checkout/payment-method-helpers.js

```diff
@@ -125,6 +125,16 @@ export async function submitStripeRedirectTransaction( paymentMethodId, transact
 	return submit( formattedTransactionData );
 }
 
+export async function submitStripeSofortTransaction( transactionData, submit ) {
```

Comment:
You should be able to get rid of this now that we have the generic one (I think this isn't being used).